specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json

```diff
@@ -5446,6 +5446,10 @@
             "$ref": "#/definitions/ImageDataDisk"
           },
           "description": "Specifies the parameters that are used to add a data disk to a virtual machine. <br><br> For more information about disks, see [About disks and VHDs for Azure virtual machines](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-windows-about-disks-vhds?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json)."
+        },
+        "zoneResilient": {
+          "type": "boolean",
+          "description":  "An optional property that specifies whether an image is zone resilient or not. Default is false. Zone resilient images can be created only in regions that provide Zone Redundant Storage (ZRS)."
```

Comment:
Just a minor comment: Don't state that this is a `property`, just start with `Specifies whether ...` 😉 We generate for tons of languages, so the concept of "properties" is not guaranteed to exist/be a convention everywhere.

Reply:
Thanks, will do!